### PR TITLE
Support enabling and disabling the low power notification via Lua

### PR DESCRIPTION
**Pull request by abcdefg30**
And only play low power notifications in Twist of Fate when the player can build buildings. Otherwise the warning is more annoying than helpful.

### CODE REVIEW

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by penev92** [comment]
FWIW this looks like a job for enabling and disabling traits via conditions, not for direct Lua manipulations (but I'm not going to block it anyway).
Also name `PlayNotification` can be really confusing. How about `IsEnabled`, since it's a property?

File: OpenRA.Mods.Common/Scripting/Properties/PowerProperties.cs [hunk 1]
```diff
@@ -43,6 +43,13 @@ public void TriggerPowerOutage(int ticks)
 		{
 			pm.TriggerPowerOutage(ticks);
 		}
+
+		[Desc("Whether the player should receive a notification when low on power.")]
+		public bool PlayLowPowerNotification
```
Comment (anvilvapre) [inline OpenRA.Mods.Common/Scripting/Properties/PowerProperties.cs:48]: Sounds a bit contradictionary. 

PlaySound, or SendOrShowNotification.

**Review by anvilvapre — APPROVED** [review]
untested.

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/fe8da5c23e7df730bab1906950cfda0cbc26c3d7...ca02a24f09d2ae7c3a3d6e8d1a3b7d6c73af89d6)

**Pull request merged**